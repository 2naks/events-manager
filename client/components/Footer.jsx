import React, { Component } from 'react';

export default class Footer extends Component {
  render() {
    return (
      <footer className="container-fluid">
        <p className="h6 text-center"><i className="fa fa-copyright" /> 2017 EVENTS MANAGER</p>
      </footer>
    );
  }
}
