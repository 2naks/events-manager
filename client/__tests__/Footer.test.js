import React, { Components } from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Footer from '../components/Footer.jsx';

Enzyme.configure({ adapter: new Adapter() });

describe('Footer should render self', () => {
  it('it should render for body', () => {
    const { wrapper } = shallow(<Footer />);
  });
});
