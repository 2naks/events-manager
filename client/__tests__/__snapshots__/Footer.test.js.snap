// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render self it should render for body 1`] = `undefined`;
